backend/jsonProcessor.js

```diff
@@ -67,11 +66,10 @@ async function processJsonFile() {
             placeId: business.place_id,
             name: placeDetails.name || business.name,
             location,
-            contactInformation:
-              placeDetails.formatted_phone_number || "Unknown",
+            contactInformation: placeDetails.formatted_phone_number || null,
             overview: placeDetails.editorial_summary
               ? placeDetails.editorial_summary.overview
-              : "Overview not provided",
+              : null,
```

Comment:
In the future for cleaner commits I'd add changes requested to the same PR; this is fine for now.